Data/Sys/Profiles/Wiimote/SDL Wii Remote + Nunchuk + GyroIR.ini

```diff
@@ -0,0 +1,40 @@
+[Profile]
```

Comment:
I think these file names would maybe be better as `with Gyro Pointing` and `with Touch Pointing` instead of `+ Gyro IR`.
"IR" is an implementation detail, and `with Pointing` matches the `Wii Remote with MotionPlus Pointing` naming.